packages/@okta/vuepress-site/guides/protect-your-api/next-steps/index.md

```diff
@@ -1,9 +1,14 @@
 ---
 title: Next Steps
 ---
-Now that you can protect your API, you may need additional information on other topics to help with other tasks.
```

Comment:
I don't think exclamation points fit our tone in the docs.

Reply:
Actually, the whole phrase "Nice job!", exclamation point or not, seems out of place.

Reply:
This was my attempt at a friendly tone, but I'm not tied to it. If you don't like it, yank it.